lib/workers/repository/update/pr/participants.ts

```diff
@@ -22,10 +31,6 @@ function filterUnavailableUsers(
     : Promise.resolve(users);
 }
 
-function noLeadingAtSymbol(input: string): string {
```

Comment:
where is this used? is the `@` already stripped somewhere?

Reply:
Yes, in two places. More precisely it is called in `addParticipants` when we `prepareParticipants` if there is at least 1 or more assignees. So actually this occurs immediately after addCodeOwners (if codeowners configured).

The same logic applies to reviewers as well.